OSD-2023.adoc

```diff
@@ -1,14 +1,13 @@
 ---
-layout: OSD-2022
+layout: OSD
 title: Open Source Day 2022
-permalink: /OSD-2022/
+permalink: /OSD/
 image:
   path: https://wildfly-security.github.io/wildfly-elytron/assets/images/posts/OpenSourceDayProject2022.png
 ---
 :page-liquid:
 
-
-Welcome Open Source Day participants! We're glad to have you join us for the hackathon.
+Welcome, Open Source Day participants! We are glad you have joined us!
```

Comment:
In this .adoc file, there's also the following lines:

```
++++
{% include OSD-2022-logo-bullets.html %}
++++
```

If you search for that file and the corresponding code in and guides.scss, you'll see that there are some circle images with logos for each operating system that are supposed to be displayed next to each operating system in step 1 in the /OSD page.

I just noticed these also aren't getting displayed in the Contribute page either, e.g., see https://wildfly-security.github.io/wildfly-elytron/contribute/. There's a bunch of white space before each operating system listed instead of the little circle images (you can see an example of the circle images here: https://wildfly-security.github.io/wildfly-elytron/guides/).

I wonder if it's just the wrong patch being specified or something small like that.

If easy to fix, that would be good to do so. Otherwise, it's ok as is.